Sources/SwiftProtobuf/Internal.swift

```diff
@@ -40,7 +40,7 @@ public enum Internal {
 
   /// Helper to loop over dictionary with values that are Messages to see if
   /// they are all initialized (see Message.isInitialized for what that means).
-  public static func areAllInitialized<K: Hashable>(_ mapToMessages: [K: Message]) -> Bool {
+  public static func areAllInitialized<K>(_ mapToMessages: [K: Message]) -> Bool {
```

Comment:
I'm surprised the compiler actually allows this, but I just checked and it does. So I guess they're inferring the `Hashable` conformance now based on the context where it's used in the argument?